samples/videoserver/pcstats.py

```diff
@@ -15,7 +15,7 @@
 
 # Dependencies
 
-from PIL import Image, ImageDraw, ImageFont     # python3 -m pip install pillow
+from PIL import Image, ImageDraw, ImageFont     # python3` -m pip install pillow`
```

Comment:
I'm guessing the addition of the back ticks happened by accident? I don't think they will work in the copy/pasted command.